src/sentry/static/sentry/app/api.jsx

```diff
@@ -191,13 +191,12 @@ export class Client {
     return this.activeRequests[id];
   }
 
-  requestPromise(path, options = {}) {
+  requestPromise(path, {includeAllArgs, ...options}) {
```

Comment:
I think you'll still need the default parameter `{}`, since the 2nd argument should be optional